### PR TITLE
samples: bluetooth: throughput: Remove logo

### PR DESCRIPTION
**Pull request by doki-nordic**
During throughput test, this sample used to print a logo to show the progress. This interfere with the test, so it is better to remove it. Now, test is time-based and it shows user expected time to end the test.

Ref: NCSDK-27222

### CODE REVIEW

**Comment by lemrey** [comment]
😢 goodbye logo

**Comment by rugeGerritsen** [comment]
This logo print was really a nice thing to have :(

**Review by FrancescoSer — APPROVED** [review]

File: samples/wifi/ble_coex/README.rst [hunk 1]
```diff
@@ -105,12 +105,9 @@ See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how
...
 
-Set up the following configuration parameters in the :file:`Kconfig.conf` file of the Bluetooth throughput sample from the :file:`nrf/samples/bluetooth/throughput` folder.
-
-* File or time-based throughput: Use :kconfig:option:`CONFIG_BT_THROUGHPUT_FILE` to select file or time-based throughput test.
-  Set it to ``n`` to enable time-based throughput test only when running Bluetooth LE throughput in central role.
-* Test duration: Use :kconfig:option:`CONFIG_BT_THROUGHPUT_DURATION` to set the duration of the Bluetooth LE throughput test only when running Bluetooth LE throughput in central role.
-  The units are in milliseconds.
+Set up the test duration configuration parameters in the :file:`Kconfig.conf` file of the Bluetooth throughput sample from the :file:`nrf/samples/bluetooth/throughput` folder.
+Use :kconfig:option:`CONFIG_BT_THROUGHPUT_DURATION` to set the duration of the Bluetooth LE throughput test only when running Bluetooth LE throughput in central role.
+The units are in milliseconds.
```
Comment (FrancescoSer) [inline samples/wifi/ble_coex/README.rst:110]:
```suggestion
Set up the test duration configuration parameters in the `Kconfig.conf` file of the Bluetooth throughput sample from the `nrf/samples/bluetooth/throughput` folder.  
Use `CONFIG_BT_THROUGHPUT_DURATION` to set the duration of the Bluetooth LE throughput test only when running Bluetooth LE throughput in the central role.  
The units are in milliseconds.
```

**Review by FrancescoSer — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Review by dchat-nordic — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]

**Comment by lemrey** [comment]
sad-crying

**Pull request merged**